Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/3UFA-nfs-target-kill/nfs-target-kill

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -x
+
+pod() {
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-openshift && bash Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/3UFA-nfs-target-kill/nfs-target-kill node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"' '"'$RELEASE_TAG'"'
+}
+
+node() {
+job_id=$(echo $1) #Gitlab job id obtain from the gitlab env ($CI_JOB_ID).
+pipeline_id=$(echo $2) # Gitlab pipeline id obtained from gitlab env ($CI_PIPELINE_ID).
+commit_id=$(echo $3) #Gitlab commit id Obtained fron gilab env ($CI_COMMIT_SHA).
+releaseTag=$(echo $4)
+source ~/.profile
+#github token to push the test result into github mayadata-io/e2e-openshift repository.
+#This token is set as an env in ~/.profile in the test cluster.
+gittoken=$(echo "$github_token")
+case_id=3UFA  #test case ID.
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+#Creating e2e custom resource result for the test.
+bash Openshift-EE/utils/pooling jobname:nfs-app-kill
+bash Openshift-EE/utils/e2e-cr jobname:nfs-target-kill jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag"
+
+################
+# LitmusBook 1 #
+################
+# Performing Application Target Chaos
+
+run_id="nfs";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=openebs-target-failure metadata=${run_id})
+echo $test_name
+
+cd litmus
+# copy the content of chaos run_litmus_test.yml into a different file to update the test specific parameters.
+cp experiments/chaos/nfs_chaos/nfs_openebs_target_failure/run_litmus_test.yml nfs_target_kill.yml
+
+sed -i -e '/name: NFS_NAMESPACE/{n;s/.*/            value: app-nfsv4-ns/}' \
+-e '/name: NFS_LABEL/{n;s/.*/            value: app=nfs/}' \
+-e '/name: NFS_PVC/{n;s/.*/            value: nfs-pvc-claim/}' \
+-e '/name: APP_LABEL/{n;s/.*/            value: name=minio/}' \
+-e '/name: APP_NAMESPACE/{n;s/.*/            value: app-minio-ns/}' nfs_target_kill.yml
+
+sed -i -e 's/value: "target-zrepl-kill"/value: "target-kill"/g' \
+-e 's/value: "cstor-volume-mgmt"/value: "cstor-istgt"/g' \
```

Comment:
change this target-kill into target-delete  and remove `-e 's/value: "cstor-volume-mgmt"/value: "cstor-istgt"/g' \
 `.

Reply:
resolved